src/pages/docs/data-folder.mdx

```diff
@@ -0,0 +1,129 @@
+
+### `extensions/`
+Extensions enhance Jan's functionality by adding new capabilities or integrating external services.
+
+## Detailed File Layout
+Each of the primary folder structures should have the following detailed structure:
+```yaml
+/assistants
+  /jan                                  
+    assistant.json                      
+  /shakespeare                          
+    assistant.json
+/engines                
+  engine_A.json                         
+/extensions
+  extensions.json                       
+  /@janhq               
+    /extension_A
+     package.json                       
+/logs
+  /app.txt
+/models
+  /model_A
+    model.json                          
+/settings
+  settings.json                         
+  /@janhq               
+    /extension_A_Settings
+     settings.json 
+/threads
+  /jan_thread_A
+   messages.jsonl                       
+   thread.json                          
+  messages.jsonl                        
+```
+
+### Assistants
+- **Default Assistant**: Located in `/assistants/jan/`, it includes an `assistant.json` configuring the default settings and capabilities.
+- **Custom Assistant Example**: The `/assistants/shakespeare/` shows a custom setup, also with its own `assistant.json`.
+
+### Engines
+- **Engine Configurations**: Files like `engine_A.json` in the `/engines/` folder configure how Jan can interact with the respective inference engines.
+
+### Extensions
+- **Extension Configuration**: The `extensions.json` in the `/extensions/` directory provides settings for all installed extensions.
+- **Specific Extensions**: Subdirectories like `/@janhq/extension_A/` contain `package.json` files which import necessary modules for each extension.
+
+### Logs
+- **Application Logs**: The file `/logs/app.txt` captures general application activity.
+
+### Models
+- **Model Configurations**: Each model directory, such as `/models/modelA/`, contains a `model.json` with settings specific to that model.
+
+### Settings
+- **General Settings**: The `settings.json` in the `/settings/` directory holds application-wide settings.
+- **Extension-specific Settings**: Additional settings for extensions are stored in respective subdirectories under `/settings/@janhq/`.
+
+### Threads
+- **Threads History**: Each thread is stored in its subdirectory, such as `/threads/jan_unixstamp/`, with files like `messages.jsonl` and `thread.json` detailing the thread settings.
+
+## Verify Installation
+To confirm that Jan has been installed correctly, follow these steps:
+
+1. Open Jan app.
+2. Locate and click on the **gear icon (⚙️)** at the bottom left corner of your screen.
+3. In the Settings menu, select **Advanced Settings**.
+4. Find the **Jan Data Folder** section and click on the **folder icon (📂)** to open the data folder.
+5. Compare the structure of your installed data folder to the expected layout data folder detailed above to ensure everything is in place.
```

Comment:
Consider moving this section here? http://localhost:3000/docs/settings#access-the-jan-data-folder

or cross linking somehow?

up to you guys to decide